rke/k8s_rke_system_images.go

```diff
@@ -1977,6 +1977,41 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.3"),
 			Nodelocal:                 m("k8s.gcr.io/k8s-dns-node-cache:1.15.7"),
 		},
+		// Reminder: Save template rancher1-1 for k8s 1.15 for Rancher v2.2.x due to WindowsPodInfraContainer image
+		// Reminder: This template contains Nodelocal image which isn't in templates before v2.3.6
```

Comment:
This might need to actually say before v2.3.7. I was confused and thought 2.3.6 should be the min version, but it should be 2.3.7

Reply:
Fixed